### DISCUSSION TITLE
ESP32 CDC (XIAO ESP32-S3) - no Display support?

### DISCUSSION THREAD

**Discussion by jwende** [arendst/Tasmota#19017] [Q&A]
Hi - I‘ve flashed the Esp32S3-CDC firmware on my Xiao board successfully  - but it seems that it doesn’t support the display. 
I can see the I2C address but if I use the command “display” it returns an error.
Do I have to create a custom build?
Thanks.
J.

**Comment by sfromis** [comment]
If what you specifically mean is `tasmota32s3cdc`, that binary does indeed not support a display.

https://github.com/tasmota/install/tree/firmware/firmware/unofficial has some `tasmota32s3` binaries including display, like `tasmota32s3-lvgl`, but without supporting CDC. Possibly the `tasmoa32s3cdc-xxxx`variants there may include display, I don't remember details.
